docs-gb/kubernetes/confluent/schema-registry.md

```diff
@@ -0,0 +1,93 @@
+---
+description: Learn how to integrate Seldon Core 2 with Schema Registry on both Confluent Cloud and Confluent Platform for having a centralized repository for schema management
+---
+
+# Schema Registry
+
+[Schema Registry](https://docs.confluent.io/platform/current/schema-registry/index.html) provides centralized schema management for data consistency and compatibility.
+
```

Comment:
We should add a line about *how* Core 2 integrates with schema registry. i.e what's the point of this feature and why would one want to leverage it.